Argo.podspec

```diff
@@ -17,5 +17,6 @@ Pod::Spec.new do |spec|
   spec.osx.deployment_target = '10.9'
 
   spec.dependency 'Runes'
+  spec.dependency 'Box'
```

Comment:
We should pin these to the Swift 1.2 versions:

`spec.dependency 'Runes', '~> 1.2'` (I think)

Box should be ~> 1.2.0 as well.